tasktiger/_internal.py

```diff
@@ -15,6 +15,14 @@
 SCHEDULED = 'scheduled'
 ERROR = 'error'
 
+# Global task context. We store this globally (and not on the TaskTiger
+# instance) for consistent results just in case the user has multiple TaskTiger
```

Comment:
It's irrelevant, since `current_task` within a task would always refer to the currently processed task.